### ISSUE TITLE
SEO Fixes - Meta Description is Missing

### ISSUE THREAD

**Issue by svikrantaxioned** [kiwix/web#195]
List of URLs with meta description missing - https://docs.google.com/spreadsheets/d/1AL3kXd3hzAivxI14DStQ2WQSmzJa6N9WWNcPkKb5QYA/edit#gid=1770667271

Description:
Pages which have a missing meta description, the content is empty or has a whitespace. This is a missed opportunity to communicate the benefits of your page and influence click through rates for important URLs.
If a meta description is completely lacking, search engines would auto-complete it, however, this is hit-and-miss, with the latter occurring more often. As part of this process, search engines might just pick the first 160 characters from your page – this could be the navigation menu or other irrelevant text. Consequently, you will lose control over what potential visitors see while browsing their query results. Given how much competition there is for user attention, this is simply unaffordable.

Resolution:
- Write unique and descriptive meta descriptions relevant for the content of the site, and highly appealing to the user on key pages to communicate the purpose of the page to users

**Comment by kelson42** [comment]
@martin8032 To me it looks like WP should force the editor to put a description, so here are my questions:
* can we set the description?
* If « yes », why are they not here? Can we make that mandatory?

**Comment by Popolechien** [comment]
@kelson42 Yes we can set descriptions (we've started, I suspect most of the pages listed have been dealt with already), and as far as I can tell there is no way to make it mandatory.

**Comment by kelson42** [comment]
The google sheet contains almost 100 entries, not sure to know what it represents in percentage, but this is significative.

**Comment by martin8032** [comment]
Content managers can set the description in the «Yoast SEO» section:

![image](https://user-images.githubusercontent.com/7620141/229312881-55a9c25d-254c-4d44-8af7-3cdfd1985ebd.png)

I don't think we can make it mandatory.
